Tests/UnitTests/Scripts/tests.js

```diff
@@ -23,16 +23,48 @@ mocha.setup({ ui: "bdd", reporter: BabylonReporter });
 const expect = chai.expect;
 
 describe("AbortController", function () {
-    it("should not throw while aborting and correctly trigger callback", function (done) {
+    it("should not throw while aborting with no callbacks", function () {
+        const controller = new AbortController();
+        expect(controller.signal.aborted).to.equal(false);
+
+        // Trigger with no callbacks
+        controller.abort();
+
+        expect(controller.signal.aborted).to.equal(true);
+    });
+
+    it("should not throw while aborting and correctly trigger both callbacks", function (done) {
         const controller = new AbortController();
         expect(controller.signal.aborted).to.equal(false);
 
         // Expect aborted to be true after abort()
         controller.signal.onabort = () => {
             expect(controller.signal.aborted).to.equal(true);
-            done();
+            // no done(); .onabort is always called first
```

Comment:
This test is relying on implementation details which is not that great.

Reply:
I'm not super proficient in the testing language, I was just introduced to the done() functionality the other day. Should the done() just be removed? Or should I separate the .onabort and .addEventListener callback tests?

Reply:
Separating would be one option. Another option is to set flags (one for each callback) and call done when all the flags are set.